Classes/Private/Manager/Database/OCTRealmManager.h

```diff
@@ -51,6 +53,7 @@
 
 - (OCTFriend *)friendWithFriendNumber:(OCTToxFriendNumber)friendNumber;
 - (OCTChat *)getOrCreateChatWithFriend:(OCTFriend *)friend;
+- (OCTCall *)getOrCreateCallWithFriendNumber:(OCTToxFriendNumber)friendNumber;
```

Comment:
Do you think it better to have both? I can add one for `getOrCreateCallWithChat`, however it would be nice to have the former since ToxAV's delegates are usually passing me the information by just friendNumber.

For example one of toxav's delegate:
`- (void)toxAV:(OCTToxAV *)toxAV callStateChanged:(OCTToxAVCallState)state friendNumber:(OCTToxFriendNumber)friendNumber`

Here I would have to do 
`OCTFriend *friend = friendWithFriendNumber(friendNumber);`
`OCTChat *chat = getOrCreateChatWithFriend:(OCTFriend *)friend;`
`OCTCall *call = getOrCreateCallWithChat:(OCTChat *)chat;``